Paralogues.pm

```diff
@@ -594,6 +759,36 @@ sub _get_transcript_from_translation {
 
 ## GET PARALOGUE VARIANTS BASED ON MATCHES OR ANNOTATION -----------------------
 
+sub _get_paralogue_vars_from_matches {
+  my ($self, $tva) = @_;
+  my $vf     = $tva->variation_feature;
+  my $allele = $tva->base_variation_feature->alt_alleles;
+
+  # get transcript for this feature (skip if not available)
+  return {} unless $tva->can('transcript');
+  my $feat   = $tva->transcript->stable_id;
+
+  my $file  = $self->{matches};
+  my $chr   = $vf->{chr};
+  my ($start, $end) = $vf->{start} < $vf->{end} ?
+    ($vf->{start}, $vf->{end}) :
+    ($vf->{end}, $vf->{start});
+
+  # get paralogue variants for this region
+  my @data = @{$self->get_data($chr, $start, $end, $file)};
```

Comment:
@nuno-agostinho,

the matches file is tabixed using the paralogous region (check L1356). But here we are querying using the variant position. For example, from the test variants for the variant `1	939112	.	G	A` the matches file will be queried using the coords `1:939112-939112` but the relevant record in the matches file -

```
1       33370577        33370579        ENST00000683057 46.4455 22.2749 1045276 1       939112  939112  G       A       ENST00000616016 NA      NA      NA      1023513 not_specified|not_provided      NA      MedGen:CN169374|MedGen:C3661900 NA      NC_000001.11:g.939112G>A        criteria_provided,_multiple_submitters,_no_conflicts    Uncertain_significance  NA      NA      single_nucleotide_variant       SO:0001483      NA      NA      SAMD11:148398   SO:0001583|missense_variant     NA      NA      NA      NA      NA      NA      NA      NA      1       751035336       NA      NA      NA      NA      NA      NA      NA
```

So, we are not getting anything. I might be missing something but currently I am not getting any output using the matches file.

Reply:
Are you querying the default options? The variants returned by default are only pathogenic (`clnsig=pathogenic`), but that variant's clinical significance is `Uncertain_significance`. To return all variants, you need to specify `clnsig=ignore`.

Reply:
I was using wrong `matches` file it seems. It did not have any paralogues for the test variant. It works now that I have updated the test variants.